packages/scripts/push-image.sh

```diff
@@ -28,12 +28,19 @@ GITHUB_SHA=$(git rev-parse --short HEAD)
 PACKAGE_NAME=$1
 PACKAGE_FOLDER=packages/$PACKAGE_NAME
 
+TAG_PREFIX=$2
+if [[ -z "$TAG_PREFIX" ]]; then
+  TAG_PREFIX=""
+else
+  TAG_PREFIX="$TAG_PREFIX-"
+fi
+
 pushd ${PACKAGE_FOLDER}
 
 # Build and push Docker images
 docker buildx build \
   --platform linux/amd64 \
-  --tag "$ECR_REPO_URI:latest" \
+  --tag "$ECR_REPO_URI:${TAG_PREFIX}latest" \
```

Comment:
how does this tag prefix work? id normally expect that we _either_ have latest (backwards compatibility) _or_ we support a tag, not a some tag suffixed with -latest